### PR TITLE
Move less compilation from client to the gradle build

### PR DESCRIPTION
**Pull request by hsablonniere**

### CODE REVIEW

File: _templates/default.template [hunk 1]
```diff
@@ -3,16 +3,8 @@
...
-    <link rel="stylesheet" type="text/css" href="/css/bootstrap.min.css" />
-    <link rel="stylesheet" type="text/css" href="/css/jquery.fancybox.css" />
-    <link rel="stylesheet/less" type="text/css" href="/css/styles.less" />
-    <link rel="stylesheet/less" type="text/css" href="/css/scroll-menu.less" />
-    <link rel="stylesheet/less" type="text/css" href="/css/sidebar-menu.less" />
-    <link rel="stylesheet/less" type="text/css" href="/css/sub-header.less" />
-    <link rel="stylesheet/less" type="text/css" href="/css/article-content.less" />
-    <link rel="stylesheet" type="text/css" href="/css/ms-com-header-footer.css">
-    <script src="/js/less.js"></script>
+    <link rel="stylesheet" type="text/css" href="/css/vendors-min.css">
```
Comment (mojavelinux) [inline _templates/default.template]: If we switch vendor to singular in the build, we'll need to update it here too.

**Comment by hsablonniere** [comment]
I applied the correction in a [fixup commit](https://robots.thoughtbot.com/autosquashing-git-commits) ;-)

**Comment by hsablonniere** [comment]
Rebased on lastest `mulesoft/staging`.

**Comment by hsablonniere** [comment]
I removed the less hacks from `scripts.js`. Let me know if I missed something.

**Comment by mojavelinux** [comment]
The only thing I see missing is that we're still using the suffix `-min.css` instead of `.min.css`. Can you update here and in the PR for the build?

**Comment by hsablonniere** [comment]
This should be merged after handling #31 and #32.

**Comment by mojavelinux** [comment]
:+1:

**Comment by mojavelinux** [comment]
The changes in this PR have been moved to #33.

**Pull request closed without merging**